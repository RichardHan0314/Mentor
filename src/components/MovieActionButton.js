import React from 'react';
import styled from 'styled-components';

const MovieActionButton = ({handleShowOptions}) => {
    return (
        <ActionIcon onClick={handleShowOptions}/>
    );
};

const ActionIcon = styled.div`
    display: flex;
    width: 30px;
    height: 30px;
    padding: 10px;
    margin: 30px 20px;
    border-radius: 50%;
    background-image: radial-gradient(circle,white 4px,transparent 5px);
    background-size: 100% 33.33%;
    background-color: #232323;
    cursor: pointer;
`;

export default MovieActionButton;
