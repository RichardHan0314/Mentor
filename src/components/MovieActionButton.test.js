import React from 'react';
import { render } from '@testing-library/react';
import MovieActionButton from './MovieActionButton';

describe('MovieActionButton component', () => {
    it('Render MovieActionButton snapshot', () => {
        const { asFragment } = render(<MovieActionButton />);
        expect(asFragment(<MovieActionButton />)).toMatchSnapshot();
    });
});