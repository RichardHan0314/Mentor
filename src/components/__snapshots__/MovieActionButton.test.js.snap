// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieActionButton component Render MovieActionButton snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm fZPbmm"
  />
</DocumentFragment>
`;
